pages/1.12/release-notes/1.12.3/index.md

```diff
@@ -0,0 +1,150 @@
+
+- DCOS-15890 - The Preflight check on advance installer shows misleading information. This release improves an error message in case Docker is not running at the start of installation.
+
+## Marathon
+- COPS-3554 - The watcher loop process monitors and re-registers (if necessary) the Marathon leader after re-election.
+
+- COPS-3593, DCOS_OSS-4193 - In previous releases, you might have services that are managed by Marathon unable to restart if the container crashes or under certain DNS failure conditions. For example, restarting services might fail if the first ZooKeeper node or first DC/OS master is unreachable. Because this problem affects high availability for Marathon, a workaround (ping zk-1) was introduced for DC/OS 1.11.5 and 1.11.6 to address the issue. In this release, the underlying issue is resolved and you can safely remove the workaround if you have it deployed. For background information about the issue and the steps to remove the workaround, see [removing the patch for Marathon failing to start if the first DC/OS is not available](https://mesosphere-community.force.com/s/article/Critical-Issue-Marathon-MSPH-2018-0004).
+
+## Mesos
+- DCOS-46554 - This change forces Mesos master to have port resources in every offer unless the offer contains disks. This helps to reduce the number of offers with no ports which are not useful for most frameworks.
+
+## Metrics
+- DCOS-47991, DCOS_OSS-4760 - In the MWST 1.13, there are gaps in various Mesos Grafana dashboards. Further investigation revealed that metrics were occasionally missing the `framework_name=<framework_name>` tag but with an additional `role_name=<framework_name>` tag. This exercise revealed a bug in the Mesos input plugin in Telegraf that may be causing this issue. The solution is to fix the hashing function to avoid potentially mapping fields to wrong tags.
+
+- DCOS_OSS-4624 - Currently, you cannot view the disk usage of a Mesos persistent volume (only Mesos sandbox is shown) for instance and the disk space used by Prometheus. This release adds the missing container metrics such as `DiskStatistics`, `Perf`, `NetTrafficControlStatistics` and `NetSNMPStatistics` from `mesos-go` and also adds all available `blkio stats`.
```

Comment:
The first sentence is a bit confusing,  Prometheus was just an example used in the ticket and  is confusing to mention here as any task using persistent volumes is impacted.   Also, 'mesos-go' is just an implementation detail. How about this instead:

"Currently, there are no container metrics that provide the disk usage of Mesos persistent volumes, on the disk usage of the Mesos sandbox is provided.   This release adds the missing container metrics such as `DiskStatistics`, `Perf`, `NetTrafficControlStatistics` and `NetSNMPStatistics` and also adds all available `blkio stats`".